clickhouse/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@clickhouse-org/click-house",
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "rsbuild dev",
+    "build": "npm run build-mf && concurrently \"npm:build:*\"",
+    "build-mf": "rsbuild build",
+    "build:cjs": "swc ./src -d dist/lib/cjs --strip-leading-paths --config-file .cjs.swcrc",
+    "build:esm": "swc ./src -d dist/lib --strip-leading-paths --config-file .swcrc",
+    "build:types": "tsc --project tsconfig.build.json",
+    "test": "cross-env LC_ALL=C TZ=UTC jest",
+    "type-check": "tsc --noEmit"
+  },
+  "main": "lib/cjs/index.js",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "devDependencies": {
```

Comment:
you can remove the dev dependencies. These are already installed by the root `package.json`